### PR TITLE
[KIECLOUD-232] - Update images of "amq-broker" to version 7.3

### PR DESCRIPTION
**Pull request by msmagnanijr**
See: https://issues.jboss.org/browse/KIECLOUD-232

Signed-off-by: Mauricio Magnani <mmagnani@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

**Comment by msmagnanijr** [comment]
> please, also update the amq-scaledown image to: amq-broker-72-scaledown-openshift:0.4

Done!

File: templates/rhpam74-authoring-ha.yaml [hunk 1]
```diff
@@ -255,7 +255,7 @@ parameters:
   displayName: Image
   name: AMQ_BROKER_IMAGE
   required: true
-  value: registry.redhat.io/amq-broker-7/amq-broker-72-openshift:1.3
+  value: registry.redhat.io/amq-broker-7-tech-preview/amq-broker-72-scaledown-openshift:0.4
```
Comment (errantepiphany) [inline templates/rhpam74-authoring-ha.yaml]: We cannot replace the broker with the scaledown controller here.

Comment (msmagnanijr) [reply templates/rhpam74-authoring-ha.yaml]: @errantepiphany sorry! my bad. I will fix that.

Comment (spolti) [reply templates/rhpam74-authoring-ha.yaml]: @msmagnanijr did you forget to update the scaledown image?
https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/master/templates/rhpam74-authoring-ha.yaml#L244-L253

**Pull request merged**